hadoop-hdds/docs/content/security/SecuringTDE.md

```diff
@@ -22,20 +22,19 @@ icon: lock
   limitations under the License.
 -->
 
-## Transparent Data Encryption
 Ozone TDE setup process and usage are very similar to HDFS TDE.
 The major difference is that Ozone TDE is enabled at Ozone bucket level
 when a bucket is created.
 
 ### Setting up the Key Management Server
 
-To use TDE, clients must setup a Key Management server and provide that URI to
+To use TDE, clients must setup a Key Management Server and provide that URI to
 Ozone/HDFS. Since Ozone and HDFS can use the same Key Management Server, this
  configuration can be provided via *hdfs-site.xml*.
```

Comment:
I think this should be core-site.xml instead of hdfs-site.xml as this is a hadoop configuration.

Reply:
Fixed